src/resources/postcss/tickets-commerce/admin/orders/single.pcss

```diff
@@ -4,6 +4,24 @@
  * @since TBD
  */
 
+/**
+ * Temporary hide
+ */
+body.wp-admin.post-type-tec_tc_order {
+	.tec-tickets-commerce-single-order--items button, .tec-tickets-commerce-single-order--items a {
+		display: none;
+	}
+	#submitdiv {
+		display: none;
+	}
+	.tec-tickets-commerce-single-order--details--item--label a {
+		display: none;
+	}
+}
+/**
+ * End Temporary hide
+ */
+
```

Comment:
All PCSS comments get compiled into the CSS (although they _should_ get removed on minification) so use sparingly.

No need to change, although I'd typically not bother with an `end` comment 😉

Reply:
I hear ya, I don't either. Just trying to wrap the "temporary" stuff so it's easy to remove later.